keyboards/planck/keymaps/oeywil/no_keycodes.h

```diff
@@ -0,0 +1,28 @@
+// øæå
+#define NO_OE   KC_SCLN
```

Comment:
These are actually already defined:
https://github.com/qmk/qmk_firmware/blob/master/quantum/keymap_extras/keymap_norwegian.h

Reply:
Do I need to change this? I have more and other symbols defined in this file than in the file you linked, so I would have to include both files.

Reply:
Well, the norwegian file also includes the nordic file, which has more:
https://github.com/qmk/qmk_firmware/blob/master/quantum/keymap_extras/keymap_nordic.h

So that may have all of what you want/need. 

But it's up to you.

Reply:
I'd like to keep my own file then.